src/js/actions/search/commands.js

```diff
@@ -60,12 +61,10 @@ define(function (require, exports) {
                 }
 
                 if (menuTree.$MENU) {
-                    resultPath += menuTree.$MENU + ">";
+                    resultPath += menuTree.$MENU + " > ";
```

Comment:
Currently, this is the only place that the `>` separator is used, so in the short term I don't think there is anything to be done here. There are multiple places in search that use separators, albeit of different types, e.g. "/" between layer groups when searching for layers, ":" between Library name and type, "    " after shortcuts.